bin/index.js

```diff
@@ -7,43 +7,49 @@ const eslintBankruptcy = require('..');
 require('hard-rejection/register');
 
 require('yargs')
-  .command('$0 <files...>', '', yargs => {
-    yargs.positional('files', {
-      describe: 'Files to modify',
-      string: true,
-      required: true
-    });
-  }, main)
+  .command(
+    '$0 <files...>',
+    '',
+    yargs => {
+      yargs.positional('files', {
+        describe: 'Files to modify',
+        string: true
+      });
+    },
+    main
+  )
   .options({
     rule: {
       alias: 'r',
       array: true,
       string: true,
-      demandOption: true,
```

Comment:
removed `demandOption` from rule, if rule is not provided - run this for *any* rule